xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -239,6 +240,47 @@ define('xwiki-lightbox', [
         $('.popover .copyImageId').parent().removeClass('hidden');
       }
     });
+    // We add the sr-only buttons to expand the lightbox
+    let toggler = $(JSON.parse($('#lightbox-config').text()).togglerTemplate);
+    lightboxImages.each(function() {
+      let lightboxToggle = toggler.clone();
+      lightboxToggle.get(0)
+        .addEventListener('focus', (e)=> {lightboxToggle.get(0).classList.remove('sr-only');});
+      lightboxToggle.get(0)
+        .addEventListener('focusout', (e) => {lightboxToggle.get(0).classList.add('sr-only');});
+      lightboxToggle.get(0)
+        .addEventListener('click', (e) => {
+        clearTimeout(showTimeout);
```

Comment:
Missing or at least misleading indentation (due to the previous line being indented).

Reply:
I liked to keep the same formatting as the other lines, but it's not clear.
Addressed in https://github.com/xwiki/xwiki-platform/pull/2281/commits/cc3d1f030adb77f0129d2a1df3b6ade3195613e9